guides/v2.2/config-guide/redis/redis-session.md

```diff
@@ -21,173 +21,44 @@ where
-</td>
-<td>2048</td>
-</tr>
-<tr>
-<td>session-save-redis-compression-lib</td>
-<td>compression_library</td>
-<td>Options: <code>gzip</code>, <code>lzf</code>, <code>lz4</code> or <code>snappy</code>.</td>
-<td>gzip</td>
-</tr>
-<tr>
-<td>session-save-redis-log-level</td>
-<td>log_level</td>
-<td>
-<p>Set to any of the following, listed in order from least verbose to most verbose:</p>
-<ul>
-<li><code>0</code> (emergency: only the most severe errors)</li>
-<li><code>1</code> (alert: immediate action required)</li>
-<li><code>2</code> (critical: application component unavailable)</li>
-<li><code>3</code> (error: runtime errors, not critical but must be monitored)</li>
-<li><code>4</code> (warning: additional information, recommended)</li>
-<li><code>5</code> (notice: normal but significant condition)</li>
-<li><code>6</code> (info: informational messages)</li>
-<li><code>7</code> (debug: the most information for development or testing only)</li>
-</ul>
-</td>
-<td>1</td>
-</tr>
-<tr>
-<td>session-save-redis-max-concurrency</td>
-<td>max_concurrency</td>
-<td>
-<p>Maximum number of processes that can wait for a lock on one session. For large production clusters, set this to at least 10% of the number of PHP processes.</p>
-</td>
-<td>6</td>
-</tr>
-<tr>
-<td>session-save-redis-break-after-frontend</td>
-<td>break_after_frontend</td>
-<td>
-<p>Number of seconds to wait before trying to break the lock for frontend (that is, storefront) session.</p>
-</td>
-<td>5</td>
-</tr>
-<tr>
-<td>session-save-redis-break-after-adminhtml</td>
-<td>break_after_adminhtml</td>
-<td>
-<p>Number of seconds to wait before trying to break the lock for an adminhtml (that is, Magento Admin) session.</p>
-</td>
-<td>30</td>
-</tr>
-<tr>
-<td>session-save-redis-first-lifetime</td>
-<td>first_lifetime</td>
-<td>
-<p>Lifetime, in seconds, of session for non-bots on the first write, or use <code>0</code> to disable.</p>
-</td>
-<td>600</td>
-</tr>
-<tr>
-<td>session-save-redis-bot-first-lifetime</td>
-<td>bot_first_lifetime</td>
-<td>
-<p>Lifetime, in seconds, of session for bots on the first write, or use <code>0</code> to disable.</p>
-</td>
-<td>60</td>
-</tr>
-<tr>
-<td>session-save-redis-bot-lifetime</td>
-<td>bot_lifetime</td>
-<td>
-<p>Lifetime, in seconds, of session for bots on subsequent writes, or use <code>0</code> to disable.</p>
-</td>
-<td>7200</td>
-</tr>
-<tr>
-<td>session-save-redis-disable-locking</td>
-<td>disable_locking</td>
-<td>
-<p>Disable session locking entirely.</p>
-</td>
-<td>0 (false)</td>
-</tr>
-<tr>
-<td>session-save-redis-min-lifetime</td>
-<td>min_lifetime</td>
-<td>
-<p>Minimum session lifetime, in seconds.</p>
-</td>
-<td>60</td>
-</tr>
-<tr>
-<td>session-save-redis-max-lifetime</td>
-<td>max_lifetime</td>
-<td>
-<p>Maximum session lifetime, in seconds.</p>
-</td>
-<td>2592000 (720 hours)</td>
-</tr>
-</tbody>
-</table>
+|Command line Parameter|Parameter name|Meaning|Default value|
```

Comment:
Markdown tables don't recognize carriage returns as line breaks, so there are several areas in this table where the returns are throwing the table formatting off. To use line breaks, you must use one or more <br> tags. For example:

`|Command line Parameter|Parameter name|Meaning|Default value|
|--- |--- |--- |--- |
|session-save-redis-host|host|Fully qualified hostname, IP address, or absolute path if using UNIX sockets.|localhost|
|session-save-redis-port|port|Redis server listen port.|6379|
|session-save-redis-password|password|Specifies a password if your Redis server requires authentication.|empty|
|session-save-redis-timeout|timeout|Connection timeout, in seconds.|2.5|
|session-save-redis-persistent-id|persistent_identifier|Unique string to enable persistent connections (for example, sess-db0).<br><br>[Known issues with phpredis and php-fpm](https://github.com/nicolasff/phpredis/issues/70").
|session-save-redis-db|database|Unique Redis database number, which is recommended to protect against data loss.<br><br>Important: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
|session-save-redis-compression-threshold|compression_threshold|Set to 0 to disable compression (recommended when [suhosin.session.encrypt = On](http://suhosin.org/stories/howtos.html#encryption-features).<br><br>Known issue with strings of more than 64KB.|2048|
|session-save-redis-compression-lib|compression_library|Options: gzip, lzf, lz4 or snappy.|gzip|
|session-save-redis-log-level|log_level|Set to any of the following, listed in order from least verbose to most verbose:<br><br>0 (emergency: only the most severe errors)<br><br>1 (alert: immediate action required)<br><br>2 (critical: application component unavailable)<br><br>3 (error: runtime errors, not critical but must be monitored)<br><br>4 (warning: additional information, recommended)<br><br>5 (notice: normal but significant condition)<br><br>6 (info: informational messages)<br><br>7 (debug: the most information for development or testing only)|1|
|session-save-redis-max-concurrency|max_concurrency|Maximum number of processes that can wait for a lock on one session. For large production clusters, set this to at least 10% of the number of PHP processes.|6|
|session-save-redis-break-after-frontend|break_after_frontend|Number of seconds to wait before trying to break the lock for frontend (that is, storefront) session.|5|
|session-save-redis-break-after-adminhtml|break_after_adminhtml|Number of seconds to wait before trying to break the lock for an adminhtml (that is, Magento Admin) session.|30|
|session-save-redis-first-lifetime|first_lifetime|Lifetime, in seconds, of session for non-bots on the first write, or use 0 to disable.|600|
|session-save-redis-bot-first-lifetime|bot_first_lifetime|Lifetime, in seconds, of session for bots on the first write, or use 0 to disable.|60|
|session-save-redis-bot-lifetime|bot_lifetime|Lifetime, in seconds, of session for bots on subsequent writes, or use 0 to disable.|7200|
|session-save-redis-disable-locking|disable_locking|Disable session locking entirely.|0 (false)|
|session-save-redis-min-lifetime|min_lifetime|Minimum session lifetime, in seconds.|60|
|session-save-redis-max-lifetime|max_lifetime|Maximum session lifetime, in seconds.|2592000 (720 hours)|`